src/data/markdown/translated-guides/es/04 Results visualization/06 InfluxDB - Grafana.md

```diff
@@ -134,6 +134,9 @@ Al subir los resultados de k6 a InfluxDB (`k6 run --out influxdb=`), puede confi
 | `K6_INFLUXDB_PASSWORD`       | Contraseña del usuario de InfluxDB                                                                                                                                                                                                                                                                                                                                                                               |         |
 | `K6_INFLUXDB_INSECURE`       | Si es verdadero, omitirá la verificación del certificado https                                                                                                                                                                                                                                                                                                                                               | `false` |
 | `K6_INFLUXDB_TAGS_AS_FIELDS` |  Un string separado por comas para establecer las métricas de k6 como [campos no indexables](https://docs.influxdata.com/influxdb/v1.8/concepts/glossary/#field) (en lugar de etiquetas). Se puede especificar un tipo opcional usando `:type` como en `vu:int` que hará que el campo sea entero. Los tipos de campo posibles son `int`, `bool`, `float` y `string`, que es el predeterminado. Ejemplo: `vu:int,iter:int,url:string,event_time:int` |         |
+| K6_INFLUXDB_PUSH_INTERVAL     | The flush's frequency of the `k6` metrics. | 1s |
+| K6_INFLUXDB_CONCURRENT_WRITES | Number of concurrent requests for flushing data. It is useful to not block the process when a request takes more than the expected time (more than flush interval). | 4 |
+
```

Comment:
Is this how it should be done @ppcano ? us adding the English and then waiting for someone to translate it to Spanish?

Reply:
@ppcano can I merge it or do you plan to provide the translation directly here?

Reply:
Please, merge it. Let's do not get blocked with the translations.